google-cloud-bigtable/.rubocop.yml

```diff
@@ -3,8 +3,6 @@ AllCops:
     - "google-cloud-bigtable.gemspec"
     - "lib/google-cloud-bigtable.rb"
```

Comment:
Did you try including `lib/google-cloud-bigtable.rb` as well?

Reply:
Feel free to disregard this scope creep and just merge, I can follow up in separate PR. LGTM and thanks!